dali/base/dasds.cpp

```diff
@@ -8681,6 +8684,7 @@ bool CCovenSDSManager::fireException(IException *e)
         }
     };
     unhandledThread.setown(new CHandleException(*this, e, restartOnError));
+#endif
     return true;
```

Comment:
Doesn't this return true need to be inside the #if (or the one above removed)?

Reply:
moved inside.